tools/nl/download_nl_files/download_nl_files.py

```diff
@@ -64,6 +64,8 @@ def main(_):
       gcs.maybe_download(index_info.embeddings_path,
                          cache_root,
                          use_anonymous_client=True)
+    path = index_info.embeddings_path
+    print(f'finished downloading {path}')
```

Comment:
Do we still need this print statement since we have [this log](https://github.com/datacommonsorg/website/blob/c774b61835bc018856b5ee7ebf3b5f6b425c68f8/shared/lib/gcs.py#L50)?

Reply:
you're right, updated